packages/flutter_tools/test/general.shard/web/chrome_test.dart

```diff
@@ -189,6 +189,25 @@ void main() {
     expect(storageDir.childFile('LOG').existsSync(), true);
     expect(storageDir.childFile('LOG').readAsStringSync(), 'contents');
   });
+
+  test('can use existing user data dir', () async {
+    final Completer<void> exitCompleter = Completer<void>.sync();
+    final Directory userDataDir = fileSystem.directory('/.tmp_rand1/flutter_tools_chrome_device.rand1');
```

Comment:
I would specify a directory that is more distinct from the random generated directories

Reply:
I'm working on this. Probably I will commit tonight.

Reply:
I could not check this item. I will check it tomorrow.

Reply:
No worries!